app/soundcloud.js

```diff
@@ -75,9 +76,11 @@ module.exports = class SoundCloud extends Events {
       titleParts = title.split(' in ', 2)
     if (titleParts.length == 2)
       // Title has " in " in it when not playing but on a playlis page
-      this.isPlaying().then(isPlaying => {
+      this.isPlaying().then(values => {
+        var isPlaying = values[0]
+        var icon = values[1]
```

Comment:
Like I said, you're the boss. I am in uncharted territory. Hope we can work together. Sorry if my messages are a bit erratic. I've had a couple brews ha ha. Cheers mate. I really want to contribute what I can to this project.